src/test/groovy/org/gradle/upgrade/wrapper/GradleWrapperUpgradePluginFuncTest.groovy

```diff
@@ -35,16 +35,17 @@ plugins {
 
 wrapperUpgrade {
     gradle {
-        'common-custom-user-data-gradle-plugin' {
-            repo = 'gradle/common-custom-user-data-gradle-plugin'
-            baseBranch = 'wrapper-upgrade-gradle-plugin-func-test-do-not-delete'
+        'wrapper-upgrade-gradle-plugin' {
+            repo = 'gradle/wrapper-upgrade-gradle-plugin'
+            baseBranch = 'func-test-do-not-delete'
+            dir = 'samples/gradle'
```

Comment:
the samples dir exists on the `func-test-do-not-delete` branch only to avoid diverging